docs/general/events.md

```diff
@@ -3,13 +3,35 @@ linkTitle: Events
 aliases: [docs/specs/semconv/general/events-general]
 --->
 
-# Semantic Conventions for Event Attributes
+# Semantic Conventions for Events
 
 **Status**: [Experimental][DocumentStatus]
 
-This document describes the attributes of standalone Events that are represented
+This document describes the characteristics of standalone Events that are represented
 in the data model by `LogRecord`s.
 
+Semantically, an Event is a named occurrence at an instant in time. It signals that
+"this thing happened at this time" and provides additional specifics about the occurrence.
+Examples of Events might include things like uncaught exceptions, button clicks, user logout,
+network connection severed, etc.
+
+In OpenTelemetry, Events are implemented as a specific type of `LogRecord` that conforms to
+the conventions included here, and Events
+[have their own API](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/event-api.md).
+The API abstracts away knowledge of `LogRecord` so that users only deal with Event
+semantics.
+
+In addition to a required name, an Event may contain a _payload_ of any type permitted by the
```

Comment:
Can we define what a "payload" is semantically? Is it the actual structural representation of the event, or an arbitrary "attachment" (a custom user-defined structure that can vary for the same event type)?

Reply:
Yes, I think this will happen as follow-up work. I think defining details of the "payload" can be handled orthogonally from where payload lives, which is what this PR is about.

Reply:
I am not sure I follow this reasoning - you want to add something to the spec without defining what it is? What difference does it make _where_ payload lives if "payload" is undefined?

Reply:
We are planning to add payloads into semconv, including rules of what they are and ways to denote their schema going forward.

This is just step one for clarity.